### PR TITLE
ezunlock: Add version 0.1

### PR DESCRIPTION
**Pull request by he3als**
A simple GUI application for deleting stuck/stale/undeletable files, open sourced alternative to IOBit Unlocker. https://github.com/demberto/EzUnlock

- [x] I have read the [Contributing Guide](https://github.com/ScoopInstaller/.github/blob/main/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by issaclin32** [comment]
`$dir\EzUnlock.exe.config` and `$dir\Helper.exe.config` stores build settings of the app. They are not involved with user config.
(This is just a reminder. No need to modify anything regard to this.)

File: bucket/ezunlock.json [hunk 1]
```diff
@@ -0,0 +1,33 @@
+{
+    "version": "v0.1",
```
Comment (issaclin32) [inline bucket/ezunlock.json:2]:
```suggestion
    "version": "0.1",
```

File: bucket/ezunlock.json [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+    "autoupdate": {
+        "architecture": {
+            "64bit":{
+                "url": "https://github.com/demberto/EzUnlock/releases/download/v$verison/EzUnlock_x64_v$version.zip",
+            },
+            "32bit":{
+                "url": "https://github.com/demberto/EzUnlock/releases/download/v$verison/EzUnlock_x64_v$version.zip",
+            }
+        }
+    }
```
Comment (issaclin32) [inline bucket/ezunlock.json:32]:
```suggestion
    "autoupdate": {
        "architecture": {
            "64bit": {
                "url": "https://github.com/demberto/EzUnlock/releases/download/v$version/EzUnlock_x64_v$version.zip"
            },
            "32bit": {
                "url": "https://github.com/demberto/EzUnlock/releases/download/v$version/EzUnlock_x64_v$version.zip"
            }
        }
    }
```

* There is a needless comma at the end of the line, which will cause **the JSON file fail to load**.
* typo: `$verison` -> `$version`

**Comment by he3als** [comment]
Thank you, surprisingly it worked in testing though

**Review by issaclin32 — APPROVED** [review]

**Pull request merged**